src/data/accessibility-quiz.ts

```diff
@@ -334,16 +343,32 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete"
   },
   {
-    Question:
-      "What is the minimum contrast ratio needed, according to the WCAG?",
+    Question: "What is the function of the `<contentinfo>` ARIA role?",
+    Answer: "displays content at that bottom of a website",
```

Comment:
I like the idea of using the footer in the answer just like MDN docs did

```suggestion
    Answer: "defines the footer content at that bottom of a website",
```